lib/index.js

```diff
@@ -31,6 +42,11 @@ var set = function (key, data, ttl, fn) {
     setParams.Expires = new Date(new Date().getTime() + ttl * 1000).toISOString();
   }
 
+  if (fromDate) {
+    setParams.Metadata = {
+      FromDate: fromDate
```

Comment:
Either pass in time and change the name of this to something more generic, OR take away passed in time and rely on AWS's modifiedDate.